src/components/ui/frame.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
```

Comment:
can you move this file to `src > common > components`.
file structure isn't super clear, my bad. the `src > components > ui` only stores the shadcdn components right now.
will clarify this in the readme going forward